src/xdocs/config.xml

```diff
@@ -242,9 +242,7 @@
 
       <source>
 &lt;module name=&quot;JavadocMethod&quot;&gt;
-  &lt;property name=&quot;severity&quot;
-               value=&quot;${checkstyle.javadoc.severity}&quot;
-               default=&quot;error&quot;/&gt;
+  &lt;property name=&quot;severity&quot; value=&quot;${checkstyle.javadoc.severity}&quot; default=&quot;error&quot;/&gt;
```

Comment:
keep ALL multiline - it is easier to read.
just change indentation.

Reply:
Jinx.

Reply:
done